vhdbuilder/packer/kube-proxy-images.json

```diff
@@ -5,16 +5,20 @@
         "downloadURL": "mcr.microsoft.com/oss/kubernetes/kube-proxy:v*",
         "amd64OnlyVersions": [],
         "multiArchVersions": [
+          "1.21.14-hotfix.20220620.1",
           "1.21.9-hotfix.20220420.2",
           "1.21.9-hotfix.20220601.1",
-          "1.21.14-hotfix.20220620.1",
-          "1.22.6-hotfix.20220330.3",
-          "1.22.6-hotfix.20220330.4",
           "1.22.11-hotfix.20220620.1",
-          "1.23.5-hotfix.20220331.2",
+          "1.22.4-hotfix.20220330.4",
```

Comment:
1.22.4 was removed?

Reply:
yes, in https://github.com/Azure/AgentBaker/commit/8168e8f7d905188a48a2ef83a3cbf4289e90cc5d

Reply:
Yes, we will be removing support for 1.22.4 (and older patch versions) in this week's RP release.

Reply:
updated and removed 1.22.4 and 1.23.3 from this PR, thanks for your confirmation @ganeshkumar5699